sys/net/transport_layer/ng_udp/ng_udp.c

```diff
@@ -39,7 +39,7 @@
 /**
  * @brief   Save the UDP's thread PID for later reference
  */
-static kernel_pid_t _pid = KERNEL_PID_UNDEF;
+kernel_pid_t ng_udp_pid = KERNEL_PID_UNDEF;
```

Comment:
I don't like this change... Feels wrong to expose more internals just for testing. Exposing this leads potentially to people miss-using this... Why don't you just use `netreg` in the test's for lookup?